sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserFacadeEjb.java

```diff
@@ -129,6 +131,50 @@
 			.collect(Collectors.toList());
 	}
 
+	@Override
+	/*
+	 * Get all users with the next higher jurisdiction, whose location contains the current users location
+	 * For facility users, this includes district and community users, if their district/community is identical with that of the facility
+	 */
+	public List<UserReferenceDto> getUsersWithSuperiorJurisdiction(UserDto user) {
+		JurisdictionLevel superordinateJurisdiction =
+			JurisdictionHelper.getSuperordinateJurisdiction(UserRole.getJurisdictionLevel(user.getUserRoles()));
+
+		List<User> superiorUsersList;
+
+		switch (superordinateJurisdiction) {
+		case NATION:
+			return null;
+		case REGION:
+			superiorUsersList = userService.getAllByRegionAndUserRoles(
+				regionService.getByReferenceDto(user.getRegion()),
+				UserRole.getWithJurisdictionLevels(superordinateJurisdiction).toArray(new UserRole[] {}));
+			break;
+		case DISTRICT:
+			// if user is assigned to a facility, but that facility is not assigned to a district, show no superordinate users. Else, show users of the district (and community) in which the facility is located
+
+			District district = null;
+			List<UserRole> superordinateRoles = UserRole.getWithJurisdictionLevels(superordinateJurisdiction);
+			if (user.getDistrict() != null) {
+				district = districtService.getByReferenceDto(user.getDistrict());
+			} else if (user.getHealthFacility() != null) {
+				district = facilityService.getByReferenceDto(user.getHealthFacility()).getDistrict();
+				superordinateRoles.addAll(UserRole.getWithJurisdictionLevels(JurisdictionLevel.COMMUNITY));
+			} else if (user.getLaboratory() != null) {
+				district = facilityService.getByReferenceDto(user.getLaboratory()).getDistrict();
```

Comment:
Laboratories don't necessarily have a district; corresponding to the change request below, I think they should just be excluded from this logic for now